package/MDAnalysis/topology/PDBParser.py

```diff
@@ -87,6 +87,44 @@ def float_or_default(val, default):
     except ValueError:
         return default
 
+digits_upper = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+digits_lower = digits_upper.lower()
+digits_upper_values = dict([pair for pair in zip(digits_upper, range(36))])
```

Comment:
rather than having two dicts, and two code paths below, could you not create a dict with both upper and lower case in it?  Ie `e` and `E` both map to whatever value?

Reply:
I am not sure this would necessarily work the same as the upper case are treated differently from the lower case (line 118 vs 124). Unless I am not seeing something you are.